src/analysis/analysis-poller.js

```diff
@@ -0,0 +1,59 @@
+var _ = require('underscore');
+var BackbonePoller = require('backbone-poller');
+
+var AnalysisPoller = {
```

Comment:
Good point. We don't have separated `vis` objects right now, but I'll change it. Thanks!